spec/models/manageiq/providers/amazon/cloud_manager/event_parser_spec.rb

```diff
@@ -1,4 +1,12 @@
 describe ManageIQ::Providers::Amazon::CloudManager::EventParser do
+  before :each do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems                 = FactoryGirl.create(:ems_amazon, :zone => zone)
+
+    allow_any_instance_of(EmsEvent).to receive(:handle_event)
```

Comment:
sure we can get rid of that if we will rewrite the code. The EmsEvent.add needs to have submethods, now it's all entangled together.